api/tests/unit/domain/usecases/get-certification-candidate-subscription_test.js

```diff
@@ -192,18 +226,79 @@ describe('Unit | UseCase | get-certification-candidate-subscription', function (
           certificationCandidateId,
           certificationBadgesService,
           certificationCandidateRepository,
+          certificationCenterRepository,
         });
 
         // then
         expect(certificationCandidateSubscription).to.deep.equal(
           domainBuilder.buildCertificationCandidateSubscription({
             id: certificationCandidateId,
             sessionId,
-            eligibleSubscriptions: [complementaryCertification1],
-            nonEligibleSubscriptions: [complementaryCertification2],
+            eligibleSubscriptions: [],
+            nonEligibleSubscriptions: [],
           })
         );
       });
     });
+
+    context('when the candidate is registered to several complementary certifications', function () {
+      context('when the candidate is eligible to only one complementary certification', function () {
+        it('should return the candidate with one eligible and one non eligible complementary certifications', async function () {
```

Comment:
Je me demande si avec ce test, on a besoin d'avoir les tests specifique:
'should return the candidate with one non eligible complementary certification'
'should return the candidate with one eligible complementary certification'